contracts/src/v0.8/automation/dev/test/BaseTest.t.sol

```diff
@@ -20,6 +20,7 @@ import {IAutomationRegistryMaster2_3} from "../interfaces/v2_3/IAutomationRegist
 contract BaseTest is Test {
   // constants
   address internal constant ZERO_ADDRESS = address(0);
+  address internal constant WRAPPED_NATIVE_ADDRESS = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
```

Comment:
can we deploy a _real_ wrapped native contract so that we can do integration testing with it?

Reply:
Hi @RyanRHall : 
1. `0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 ` is the [WETH](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code) contract. When you say `real`, do you mean we copy the WETH contract and deploy our own? 

2. How does integration test in solidity work? Is the integration test in foundry?